packages/qwik-city/runtime/src/index.ts

```diff
@@ -58,10 +58,12 @@ export { routeAction$, routeActionQrl } from './server-functions';
 export { globalAction$, globalActionQrl } from './server-functions';
 export { routeLoader$, routeLoaderQrl } from './server-functions';
 export { server$, serverQrl } from './server-functions';
-export { zod$, zodQrl } from './server-functions';
+export { zod$, zodQrl, valibot$, valibotQrl } from './server-functions';
 export { validator$, validatorQrl } from './server-functions';
 
 export { z } from 'zod';
+import * as v from 'valibot';
+export { v };
```

Comment:
I don't think we should be exporting `valibot` like this. (I know we do that with `z` but `z` is a specific object whereas `v` is a module. I don't think we should export anything, and people can directly import from `valibot` to get the right things.

Reply:
Worried about the same thing myself. Removed the re-export.